src/Analysis/Ast/Impl/Values/Collections/PythonCollection.cs

```diff
@@ -72,5 +75,7 @@ internal class PythonCollection : PythonInstance, IPythonCollection {
                     return 0;
             }
         }
+
+        public bool IsExact { get; protected set; }
```

Comment:
Why `protected set`? Looks like we only set it in constructor

Reply:
I made it have the same level as `Contents` did, so it could be set in the same places, but that one seems like it could drop `set` as well without issue if you wanted me to make that change.

Reply:
Contents needed `protected set`, but I've dropped `protected set` from `IsExact` (but we can add it back later if we come up with some more interesting use case).